build/pipeline.yml

```diff
@@ -0,0 +1,53 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+pr: none
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: microsoft/vscode-engineering
+      ref: main
+      endpoint: Monaco
+
+parameters:
+  - name: publishPackage
+    displayName: 🚀 Publish vscode-markdown-languageservice
+    type: boolean
+    default: false
+
+extends:
+  template: azure-pipelines/npm-package/pipeline.yml@templates
+  parameters:
+    npmPackages:
+      - name: vscode-markdown-languageservice
+
+        buildPlatforms:
+          - name: Linux
+            nodeVersions:
+              - 16.x
+          - name: MacOS
+            nodeVersions:
+              - 16.x
+          - name: Windows
+            nodeVersions:
+              - 16.x
+
+        buildSteps:
+          - script: npm install
+            displayName: Install dependencies
+
+          - script: npm run compile
```

Comment:
I see we run both `npm run compile` and `yarn compile`. Is that necessary?

Reply:
No, good catch!